src/main/java/org/folio/circulation/domain/notice/NoticeDescriptor.java

```diff
@@ -0,0 +1,115 @@
+package org.folio.circulation.domain.notice;
+
+import org.joda.time.Period;
+
+public class NoticeDescriptor {
```

Comment:
What is a notice descriptor?

Reply:
A record in notice policy describing how notice should be sent for specific event.

Reply:
How does Darcy refer to this, is it using this name?

I'm trying to understand if there is a domain terminology for this.

Reply:
Judging from what we have on UI, it is called just 'Notice'. 
![image](https://user-images.githubusercontent.com/42538529/54218842-fdab0d00-44f6-11e9-812c-889508d339b8.png)

Reply:
Can be called something like 'NoticeSetting'. I'm not really sure about the naming.

Reply:
I’d prefer something like `NoticeSetting` or `NoticeConfiguration`.

I might even suggest notice if that is what it is called in the domain, however that is likely ambiguous with the actual notice itself. So maybe that is a conversation for another day.

Reply:
Renamed to NoticeConfiguration